test/ProblemDetailsNotFoundHandlerTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-problem-details for the canonical source repository
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-problem-details/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendTest\ProblemDetails;
+
+use ErrorException;
+use Interop\Http\ServerMiddleware\DelegateInterface;
+use PHPUnit\Framework\TestCase;
+use Prophecy\Argument;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Message\StreamInterface;
+use Zend\ProblemDetails\ProblemDetailsNotFoundHandler;
+use Zend\ProblemDetails\ProblemDetailsResponseFactory;
+
+class ProblemDetailsNotFoundHandlerTest extends TestCase
+{
+    use ProblemDetailsAssertionsTrait;
+
+    public function acceptHeaders()
```

Comment:
In the other test case (`ProblemDetailsNotFoundHandlerFactoryTest`) on all methods you have declared return type, but not in this test case. Can we add it please, for consistency? Thx 😄